stdlib/public/TensorFlow/TensorShape.swift

```diff
@@ -159,3 +159,10 @@ extension TensorShape : Codable {
     self.init(dimensions)
   }
 }
+
+extension TensorShape : CustomStringConvertible {
+  @inlinable
+  public var description: String {
+    return "TensorShape(\(dimensions))"
```

Comment:
Since we think that `[2, 2]` is more ideal, calling `dimensions.description` should be sufficient.

```suggestion
    return dimensions.description
```